packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -24,143 +24,65 @@ $bpk-spacing-v2: true;
   padding: 0 bpk-spacing-base();
   align-items: center;
   border: none;
+  border-radius: $bpk-border-radius-xs * 2;
 
-  @include bpk-border-radius-pill;
+  &--has-trailing {
+    padding-right: bpk-spacing-md();
+  }
```

Comment:
When BpkSelectableChip with trailingAccessoryView, padding-right should be 8px.

Reply:
Perhaps you could place this under `&__trailing-accessory-view` instead of creating a new class, wdyt?